ruff.toml

```diff
@@ -0,0 +1,17 @@
+# Enable flake8/pycodestyle (`E`), Pyflakes (`F`), flake8-bandit (`S`), and isort (`I`) rules.
+select = ["E", "F", "S", "I"]
+fixable = ["I"]
+
+# Never enforce `E402`, `E501` (line length violations),
```

Comment:
So the main issue is that black does not apply line-length fixes to comments while flake8/ruff will flake comments that exceed this length. 

The thought is also that black will fix the line length violations anyways since it will be run before flake8/ruff.

Reply:
If its just comments that will get picked up in addition then I think this could be okay :thinking: ? It would be nice to have them both compatible.

Granted imo line length is an opinion for the formatter and should be left to the formatter.

But Akshay has some good points about development environments.

Reply:
Decided that it would be best to leave these checks out of this PR but add them back in as part of a new PR. Created an issue to track this: https://github.com/NVIDIA/modulus/issues/172